server/routes/ad.ts

```diff
@@ -314,6 +315,32 @@ const searchDetector = async (
   }
 };
 
+const searchResults = async (
+  req: Request,
+  h: ResponseToolkit,
+  callWithRequest: CallClusterWithRequest
+): Promise<ServerResponse<any>> => {
+  try {
+    //@ts-ignore
+    const requestBody = JSON.stringify(req.payload);
+    const response = await callWithRequest(
+      req,
+      'ad.searchResults',
+      { body: requestBody }
+    );
+    return {
+      ok: true,
+      response,
+    };
+  } catch (err) {
+    console.log('Anomaly detector - Unable to search anomaly result', err);
+    if (isIndexNotFoundError(err)) {
+      return { ok: true, response: { totalDetectors: 0, detectors: [] } };
```

Comment:
minor: this response doesn't matter since the reducer isn't using the responses right now, but this response doesn't make a lot of sense. Maybe just return empty for now and add TODO

Reply:
Good catch, will fix this in next PR.